mlflow/store/hdfs_artifact_repo.py

```diff
@@ -0,0 +1,124 @@
+import os
+
+from hdfs3 import HDFileSystem
+from six.moves import urllib
```

Comment:
please consider lazy loading hdfs3  - there is a common practice to delay import till the  __init__

Reply:
Included lazy loading corrections for hdfs3